authors/apps/authentication/renderers.py

```diff
@@ -23,3 +23,25 @@ def render(self, data, media_type=None, renderer_context=None):
         return json.dumps({
             'user': data
         })
+
+class SignupUserJSONRenderer(JSONRenderer):
```

Comment:
hey @Bakley Why create another renderer? I can see it's the same as the `UserJSONRenderer` You have only added the message
Why not update the existing renderer and use it as it is?

Reply:
Hey, @graycadeau sorry I saw this review this morning. The reason for having another render was because the UserJSONRender is used on multiple view classes and  I wanted a specif renderer one which has a special message for the Basic Signup to pass to the user that an email has been sent to the email address. The main reason was to get the message as its own key. The best way we saw was to create a different render.
<img width="1126" alt="Screenshot 2019-05-02 at 17 45 53" src="https://user-images.githubusercontent.com/7009215/57122300-49569600-6d85-11e9-842d-38bf1477cccc.png">

Reply:
@Bakley Since every user will receive the same message upon registration, you can return the message as part of the response under RegistrationAPIView, and you will not need this renderer.

Reply:
@graycadeau you mean something like this?
<img width="1120" alt="Screenshot 2019-05-02 at 17 15 56" src="https://user-images.githubusercontent.com/7009215/57124452-91c68180-6d8e-11e9-9953-820f2133215c.png">